Index: sfPropelDatabase.class.php
===================================================================
--- sfPropelDatabase.class.php	(revision 4128)
+++ sfPropelDatabase.class.php	(working copy)
@@ -86,12 +86,13 @@
           'phptype'    => $this->getParameter('phptype'),
           'hostspec'   => $this->getParameter('hostspec'),
           'database'   => $this->getParameter('database'),
-          'username'   => $this->getParameter('username'),
+          'user'       => $this->getParameter('username'),
           'password'   => $this->getParameter('password'),
           'port'       => $this->getParameter('port'),
           'encoding'   => $this->getParameter('encoding'),
           'persistent' => $this->getParameter('persistent'),
           'protocol'   => $this->getParameter('protocol'),
+          'dsn'   => $this->getParameter('dsn')
         ),
       );
   }
